src/types/annotator.js

```diff
@@ -13,6 +18,36 @@
  * @prop {DocumentMetadata} document
  */
 
+/**
+ * An object representing the location in a document that an annotation is
+ * associated with.
+ *
+ * @typedef Anchor
+ * @prop {AnnotationData} annotation
+ * @prop {HTMLElement[]} [highlights]
```

Comment:
I literally did not even know until this moment that `highlight`s were elements—they've been so abstract to me. Heh.

But...having clued myself in thusly...what conditions would lead to multiple highlight elements per anchor?

Reply:
> But...having clued myself in thusly...what conditions would lead to multiple highlight elements per anchor?

Highlight elements wrap one or more text nodes. If the selection spans text across multiple _elements_ (ie. so the text nodes are not all siblings of one another) then there will be multiple highlight nodes.